packages/react-next/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -110,19 +57,44 @@ export class ChoiceGroupBase extends React.Component<IChoiceGroupProps, IChoiceG
       optionsContainIconOrImage: options.some(option => !!(option.iconProps || option.imageSrc)),
     });
 
-    const labelId = this._id + '-label';
-    const ariaLabelledBy = this.props.ariaLabelledBy || (label ? labelId : this.props['aria-labelledby']);
+    const ariaLabelledBy = props.ariaLabelledBy || (label ? labelId : props['aria-labelledby']);
+
+    const [keyChecked, setKeyChecked] = useControllableValue(props.selectedKey, defaultSelectedKey);
+
+    const [keyFocused, setKeyFocused] = React.useState<string | number>();
+
+    useComponentRef(props, keyChecked, id);
+
+    const onFocus = (ev: React.FocusEvent<HTMLElement>, option: IChoiceGroupOption) => {
+      setKeyFocused(option.optionKey);
+    };
+
+    const onBlur = (ev: React.FocusEvent<HTMLElement>, option: IChoiceGroupOption) => {
+      setKeyFocused(undefined);
+    };
+
+    const onOptionChange = (evt: React.FormEvent<HTMLElement | HTMLInputElement>, option: IChoiceGroupOption) => {
+      const { onChange } = props;
+
+      setKeyChecked(option.optionKey);
+
+      if (onChange) {
+        onChange(
+          evt,
+          find(props.options || [], (value: IChoiceGroupOption) => value.optionKey === option.optionKey),
+        );
+      }
+    };
+
+    if (process.env.NODE_ENV !== 'production') {
+      warnMutuallyExclusive('ChoiceGroup', props, {
+        selectedKey: 'defaultSelectedKey',
+      });
+    }
 
-    // TODO (Fabric 8?) - if possible, move `root` class to the actual root and eliminate
-    // `applicationRole` class (but the div structure will stay the same by necessity)
     return (
-      // eslint-disable-next-line deprecation/deprecation
-      <div className={classNames.applicationRole} {...divProps}>
-        <div
-          className={classNames.root}
-          role="radiogroup"
-          {...(ariaLabelledBy && { 'aria-labelledby': ariaLabelledBy })}
-        >
+      <div className={classNames.root} {...divProps} ref={forwardedRef}>
```

Comment:
We need to document the move of the `root` class in RELEASE_NOTES.md breaking changes.

Also, since the `applicationRole` class is no longer being used, we should remove it from `IChoiceGroupStyles` (and also document that breaking change).